<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>version_content_change</class>
 <widget class="QDialog" name="version_content_change">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Changes to Version</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>257</width>
     <height>136</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="bookNameLineEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="bookNameLabel">
          <property name="text">
           <string>Book Name: </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="seasonNameLabel">
          <property name="text">
           <string>Season Name: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="seasonNameLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="versionNumberLabel">
          <property name="text">
           <string>Version Number</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="versionNumberSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
